src/msg/simple/Accepter.cc

```diff
@@ -354,12 +392,29 @@ void *Accepter::entry()
     } else {
       ldout(msgr->cct,0) << __func__ << " no incoming connection?  sd = " << sd
 	      << " errno " << errno << " " << cpp_strerror(errno) << dendl;
-      if (++errors > 4)
-	break;
+      if (++errors > 4){
```

Comment:
I know that the magic number existed before, but we should make both this "4" and the sleep time be configurable.

Reply:
Yes, it's better to make it configurable than showing some magic number.
I will modify the magic number to be configurable.